packages/blockchain-wallet-v4-frontend/src/data/experiments/sagas.ts

```diff
@@ -0,0 +1,19 @@
+import { call, put } from 'redux-saga/effects'
+
+import { actions as A } from './slice'
+
+export default ({ api }) => {
+  const fetchExperimentsSaga = function* () {
+    try {
+      yield put(A.fetchLoading())
+      const experiements: ReturnType<typeof api.getExperiments> = yield call(api.getExperiments)
```

Comment:
We recently started using `typed-redux-saga`. Just import `call` from it instead to avoid type casting:

```suggestion
      const experiements = yield* call(api.getExperiments)
```

Reply:
Is that `yield*` or just `yield`? Getting an error with the `*`

Reply:
Did you import `call` from `typed-redux-saga`?